routes/api/users/index.js

```diff
@@ -54,4 +54,17 @@ router.get('/:username', async (req, res) => {
     }
 });
 
+/**
+ * Check if email is taken
+ */
+router.get('/:email/exists', async (req, res) => {
+   const email = req.params.email;
```

Comment:
You may want to make this `const email = req.params.email || ''` to catch errors if they don't supply an email